lib/core/index.js

```diff
@@ -6,7 +6,8 @@ var axe = axe || {};
 axe.version = '<%= pkg.version %>';
 
 if (typeof define === 'function' && define.amd) {
-	define([], function () {
+	// Explicitly naming the module to avoid mismatched anonymous define() modules when injected in a page
+	define('aXe', [], function () {
```

Comment:
This should probably be `axe-core`, not `aXe`, since that way it matches how to load it with `require()`